pkg/microservice/packager/core/service/service.go

```diff
@@ -94,55 +95,53 @@ func buildTargetImage(imageName, imageTag, host, nameSpace string) string {
 	return ret
 }
 
+func ExtractErrorDetail(in io.Reader) error {
+	dec := json.NewDecoder(in)
+	for {
+		var jm jsonmessage.JSONMessage
+		if err := dec.Decode(&jm); err != nil {
+			if err == io.EOF {
```

Comment:
Will `Decode()` return error of `io.EOF`? https://github.com/golang/go/issues/25956